components/TARGET_PSA/services/crypto/COMPONENT_SPE/psa_crypto_access_control.c

```diff
@@ -28,16 +28,19 @@ typedef struct psa_crypto_access_control_s {
 
 static psa_crypto_access_control_t crypto_access_control_arr[PSA_KEY_SLOT_COUNT];
 
-#define PSA_CRYPTO_ACCESS_CONTROL_RESET() (memset(crypto_access_control_arr, 0, sizeof(crypto_access_control_arr)))
+static inline void psa_crypto_access_control_reset()
+{
+    memset(crypto_access_control_arr, 0, sizeof(crypto_access_control_arr));
```

Comment:
isn't it 
psa_crypto_access_control_reset();?

Reply:
The macro was removed in 6475bc65d818fc9ff4fc6c2c02de3d1b7ecde24a in favor of the static inline function.